<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Message</class>
 <widget class="QWidget" name="Message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Message{border-image:url(:/new/prefix1/Picture/sea3.jpg);}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout"/>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>0</y>
     <width>381</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{border-image:url(:/new/prefix1/Picture/sea3.jpg);}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QToolButton" name="Head">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>-11</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="styleSheet">
         <string notr="true">#nameEdit{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border:1px;
            border-radius:20px; 
            padding:2px 4px;  
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>139</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>109</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID：</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="IDEdit">
        <property name="styleSheet">
         <string notr="true">#IDEdit{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border:1px;
            border-radius:20px; 
            padding:2px 4px;  
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
